clang/test/C/C2y/n3369.c

```diff
@@ -144,3 +177,13 @@ void test_compound_literals() {
   static_assert(_Countof((int[2]){}) == 2);
   static_assert(_Countof((int[]){1, 2, 3, 4}) == 4);	
 }
+
+static int test_f1();
+static int test_f2();  // expected-warning {{never defined}}
```

Comment:
Pretty sure this would be `function 'test_f2' has internal linkage but is not defined`

Reply:
> It seems it's not triggering anything at all. :|

Yeah, I would turn that comment into a FIXME about how we *should* get the diagnostic but it's a bug that we don't.

Reply:
done.